src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!-- NuGet -->
-    <Version>1.19.3</Version>
+    <Version>1.19.6-prerelease</Version>
```

Comment:
You should revert this change, that way when this PR and PR #987 are both merged, only that one modifies the file

Reply:
I don't follow. We still want the other version attributes to get bumped here even if the other PR keeps the assembly version at 1.0.0. If I revert this it will stay at `1.19.3` even with the other PR. Did I miss something?

Reply:
i.e. we want both changes

Reply:
Ah nevermind, you're right we do want both changes. My bad 🙃